src/labels/CustomLabels.labels

```diff
@@ -9519,6 +9519,16 @@ Once the process is 100% complete, you can safely leave or refresh this page.</v
         <shortDescription>stgHelpEnableSoftCreditRollups</shortDescription>
         <value>When selected, rolls up Closed/Won Opportunity totals to Contact records based on assigned Opportunity Contact Roles. Soft credit rollups only happen in nightly batches, but you can run them manually from the Rollup Donations Batch settings.</value>
     </labels>
+    <labels>
+        <fullName>stgHelpEnhancedRDBatch</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
```

Comment:
- Protected should be `true` for every new label.
- Short description can be more descriptive: `Page description displayed when enhanced RD batch is to be run` or similar. @sam-knox